utensor_cgen/backend/operators.py

```diff
@@ -201,6 +243,26 @@ def __init__(self, op_info, **kwargs):
                                              x_dtype, w_dtype, out_dtype, 
                                              ref_counts, to_eval)
 
+@OperatorFactory.register
+class _ReluOperator(_Operator):
+
+  op_type = "Relu"
+
+  def __init__(self, op_info, **kwargs):
+    _Operator.__init__(self)
+    inputs = [tensor_info.name for tensor_info in op_info.input_tensors]
+    output = op_info.output_tensors[0].name
+    in_dtype, out_dtype = (op_info.input_tensors[0].dtype,
+                            op_info.output_tensors[0].dtype)  #NT: why separate this out?
```

Comment:
Oh, you can remove the comments by @neil-tan and I.
@neil-tan asked me this in the comment before, so I have to answered it in the code. Haha